src/table.js

```diff
@@ -492,6 +475,24 @@ export function __table(source, operations) {
     const [{value: column}] = operands;
     const values = operands.slice(1).map(({value}) => value);
     switch (type) {
+      // valid (matches the column type)
+      case "v": {
+        const [colType] = values;
+        const filter = colType === "date" ?
+          (d) => (d[column] instanceof Date) && !Number.isNaN(+d[column]) :
+          (d) => typeof d[column] === colType;
+        source = source.filter(filter);
+        break;
+      }
+      // not valid (doesn't match the column type)
+      case "nv": {
+        const [colType] = values;
+        const filter = colType === "date" ?
+          (d) => !(d[column] instanceof Date) || Number.isNaN(+d[column]) :
+          (d) => typeof d[column] !== colType;
```

Comment:
Can we have a shared function for `v` so that we can call that here and negate it? There are a number of subtleties (especially given the other comment) so it’d be good to not duplicate the logic.

Reply:
Done! Added `isValid`